libraries/Microsoft.Bot.Expressions/Expression.cs

```diff
@@ -100,6 +100,16 @@ public Expression(ExpressionEvaluator evaluator, params Expression[] children)
         /// </value>
         public ReturnType ReturnType => Evaluator.ReturnType;
 
+        /// <summary>
+        /// Parse an string into an expression object.
+        /// </summary>
+        /// <param name="expression">expression string.</param>
+        /// <returns>resulting expression.</returns>
+        public static Expression Parse(string expression)
+        {
+            return new ExpressionEngine().Parse(expression);
```

Comment:
>new ExpressionEngine() [](start = 19, length = 22)

Given this is static why wouldn't you just use a static engine?

Reply:
ExpressionEngine doesn't actually have any properties, it's just one member which by default is the static ExpressionFunctions table.

---
In reply to: [376564136](https://github.com/microsoft/botbuilder-dotnet/pull/3350#discussion_r376564136) [](ancestors = 376564136)